docker-compose.yml

```diff
@@ -3,10 +3,7 @@ version: '3.7'
 services:
 
     postgres:
-        platform: linux/amd64
-        build:
-            context: ./
-            dockerfile: ./docker/postgres/Dockerfile
+        image: "postgis/postgis:13-3.2-alpine"
```

Comment:
This is a good change 👍 
I was pondering if the database version could be 14 series inline with the envs - but nevertheless this is good.

Reply:
Yes, perhaps it would not be too bad to have the 14.11 version here (I think Platta uses 14.9 or so). I'll see if the same version that Platta uses is available as a combination with PostGIS.

Reply:
The PostGIS tag "14-3.2" actually gives PostgreSQL version 14.5 instead of 14.11. Must have been a later tag that I tested previously. Anyways, the version was just bumped.